### PR TITLE
Add Universal Module Definition to be able to use it with RequireJS

### PR DESCRIPTION
**Pull request by slgal**
Hello, I think it will be nice to to add Universal Module Definition. So knockout-reactor can be used with RequireJS and other module loaders.
More here:  https://github.com/umdjs/umd

Thanks

### CODE REVIEW

File: src/knockout.reactor.js [hunk 1]
```diff
@@ -2,7 +2,18 @@
...
+    if (typeof require === 'function' && typeof exports === 'object' && typeof module === 'object') {
+        factory(require('knockout'));
+        // AMD
+    } else if (typeof define === 'function' && define.amd) {
+        define(['knockout'], factory);
+        // Normal script tag
+    } else {
+        factory(window.ko);
+    }
+}(function (ko, $) {
```
Comment (bago) [inline src/knockout.reactor.js]: Why is $ here?

Comment (slgal) [inline src/knockout.reactor.js]: agree, is my mistake

Comment (slgal) [inline src/knockout.reactor.js]: I don't know how to update that PR so I  just pushed the fix to my fork.

Comment (bago) [inline src/knockout.reactor.js]: pushing to your fork is the "right way": the PR has already been updated.

**Pull request merged**